app/backend/src/couchers/models.py

```diff
@@ -346,6 +347,13 @@ def display_last_active(self):
         """
         return last_active_coarsen(self.last_active)
 
+    @property
+    def user_link(self):
+        """
+        Link to this user's profile
+        """
+        return urls.user_link(self.username)
+
```

Comment:
Move this helper function to email generation. Backend database has nothing to do with urls.

Reply:
Sure, I'll inject this in another way into the templates